README.rst

```diff
@@ -5,52 +5,132 @@ plone.staticresources
 Static JS and CSS resources for Plone.
 
 
-This add on contains all JavaScript and CSS resources from Plone.
-Prior to Plone 5.2, they were located in Products.CMFPlone.
+This add on contains all JavaScript and CSS resources used by Plone.
+
+Prior to Plone 5.2, they were located in ``static/`` directory of
+``Products.CMFPlone`` package.
+
+Due to the specific tooling and workflows used to build frontend resources, and
+also the different maintenance and release needs of ``Products.CMFPlone``
+itself, Plone static resources are now distributed in this package. For a
+complete list of reasons, read the§ `PLIP
+1653<https://github.com/plone/Products.CMFPlone/issues/1653>`_.
 
 .. note::
-  A note on yarn: We require the use of yarn, because yarn supports installing the packages in a custom directory, which npm doesn't.
-  Our package directory is located at: ``src/plone/staticresources/static/components``.
 
+  A note on yarn: We require the use of yarn, because it supports installing
+  packages in a custom named directory, which npm doesn't (even using
+  ``--prefix``, ``node_modules`` is hardcoded). Our package directory is
+  located at: ``src/plone/staticresources/static/components``.
+
+
+Target audience
+---------------
+
+This documentation aims at:
+
+Integrators and Developers
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Who want to customize and/or maintain the frontend of a Plone site for their
+customers:
+
+- upgrade existing versions or install new ``npm`` packages
+- integrate new JavaScript/CSS behavior, eventually wrapped in a ``pattern``
+for optimal integration and reuse within Plone frontend ecosystem
+
+
+Themers
+^^^^^^^
+
+Who want to change the look and feel of a Plone site and need to:
+
+- understand how to correctly bundle changes made into existing or new frontend
+code
+-
```

Comment:
I'll mark it with XXX, this is a working in progress at the moment mostly between me and @thet, and as I said before you're as much as welcome to fill in the gaps. It's not the perfect workflow but I believe it will allows us to get things done faster than with more indirect approaches like Github PR comments. I'll make sure to remove these all out before our first release.